packages/css-library/src/stylesheets/print.scss

```diff
@@ -0,0 +1,29 @@
+.print-only {
+  display: none;
+}
+
+@media print {
+  header, footer, nav {
```

Comment:
A print stylesheet! How do we feel about adding this? There was a mix of CSS elements and classes and instead of splitting them up, I figured they could all just be together. 

Here is the original in Formation: https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/master/packages/formation/sass/modules/_m-print.scss

Reply:
I'll have to read up on `print` lol. I'm inclined to keep it separate because it feels like a pretty specific use case and does some minor things to the `.schemaform-` classes, but it's a weak opinion.

Reply:
@micahchiang Now that you mention `.schemaform-`, should that even be in Formation or CSS Library? Wouldn't that be better in the forms library itself?

```css
  .schemaform-title,
  .schemaform-subtitle,
  .screen-only {
    display: none;
  }
```

Reply:
Nevermind! It seems like it's in many different apps so probably a legit global need.

![Screenshot 2023-12-13 at 10 44 15 AM](https://github.com/department-of-veterans-affairs/component-library/assets/872479/541115f7-67eb-411e-bf86-94b716b9b440)

Reply:
yeah all of those schemaform- classes are gonna be an interesting thing to deal with. Their use is very pervasive to say the least.